File: frontend_mycoffeedau/src/models/review.ts
export type Review = {
  id: number;
  idUser: string;   // Đổi từ id_user thành idUser
  idDrink: string;  // Đổi từ id_drink thành idDrink
  avatar: string;
  userName: string; // Đổi từ username thành userName
  rating: number;
  comment: string;
  dateReview: string;
};

File: backend_mycoffeedau/backend_mycoffeedau/Model/Review.cs
namespace backend_mycoffeedau.Models
{
    public class Review
    {
        public int Id { get; set; }
        public string IdUser { get; set; } = string.Empty;
        public string IdDrink { get; set; } = string.Empty;
        public int Rating { get; set; } // 1-5 sao
        public string Comment { get; set; } = string.Empty;
        public string Avatar { get; set; } = string.Empty;
        public string UserName { get; set; } = string.Empty;
        public DateTime DateReview { get; set; } = DateTime.Now;
    }
}